src/ralph/cli.py

```diff
@@ -32,6 +34,8 @@
 @click_log.simple_verbosity_option(get_root_logger())
 def cli():
     """Ralph is a stream-based tool to play with your logs"""
+    if SENTRY_DSN is not None:
+        sentry_sdk.init(SENTRY_DSN, traces_sample_rate=1.0)
```

Comment:
I would recommend adding more configurations as:

- the `release`
- the `environment`

Reference documentation: https://docs.sentry.io/platforms/python/configuration/options/

Reply:
Indeed, I added them